logging/zap/grpclogger.go

```diff
@@ -44,3 +45,80 @@ func (l *zapGrpcLogger) Printf(format string, args ...interface{}) {
 func (l *zapGrpcLogger) Println(args ...interface{}) {
 	l.logger.Info(fmt.Sprint(args...))
 }
+
+// GrpcLoggerV2Options allows you to set grpclogger options when using ReplaceGrpcLoggerV2.
+type GrpcLoggerV2Options struct {
+	verbosity int
+}
+
+// DefaultGrpcLoggerV2Options is a set of default options for ReplaceGrpcLoggerV2.
+var DefaultGrpcLoggerV2Options = &GrpcLoggerV2Options{}
```

Comment:
remove this and GrpcLoggerV2Options once done the below comments :+1:

Reply:
done